etc/standardnotes-desktop.profile

```diff
@@ -0,0 +1,25 @@
+# Firejail profile for standardnotes-desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/standardnotes-desktop.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
```

Comment:
Can we try to add
```
include /etc/firejail/disable-devel.inc
include /etc/firejail/disable-interpreters.inc
```
As well?

The disable-interpreters one is a bit tricky to get right the first time. If something breaks, just post the terminal output here and we'll try to get it working. :smile: